### PR TITLE
satellite/console: create owner_id column for project table

### PR DESCRIPTION
**Pull request by crawter**
What: 
    1.Column owner_id should be added to projects table
    2. Projects without users will be removed 
    3. bucket_metainfos that refer to projects with owner_id == null will also be deleted.
    3.OwnerID column of rest projects will be filled with id of first project member

So, also, i have updated all previous migrations. The main reason of it - that we have 2 projects in all our testdata files.

One of them has a project member. second not. 

And due to one of my migration steps - project without a project member should be deleted.

But a lot of tables (api keys, bucket_metainfos, project_invoice_stamps, project_members, project_payments) are referencing to a project table. 

So, some of INSERT statements ref to a project with project member, some - to projects that would be deleted. 

Not to delete all this insertions i have added a project member to all previous testdata files.

Why: Our new payment flow requires this changes, because only project owner is responsible for payment

Please describe the tests:  Tests updated to check ownerID field
 
Please describe the performance impact: none

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/satellitedb/migrate.go [hunk 1]
```diff
@@ -1081,6 +1081,35 @@ func (db *DB) PostgresMigration() *migrate.Migration {
...
+					`ALTER TABLE projects
+					    ADD COLUMN owner_id BYTEA;`,
+
+					`UPDATE projects as proj
+						SET owner_id =
+							(SELECT member_id
+								FROM project_members
+									WHERE project_id = proj.id
+										ORDER BY created_at ASC
+											LIMIT 1);`,
```
Comment (egonelbre) [inline satellite/satellitedb/migrate.go]: This indentation looks really weird. Maybe, something like:

```
UPDATE projects
SET owner_id = (
	SELECT member_id
	FROM project_members
	WHERE project_id = projects.id
	ORDER BY created_at ASC
	LIMIT 1
);`,
```

File: satellite/satellitedb/testdata/postgres.v50.sql [hunk 2]
```diff
@@ -305,6 +305,7 @@ INSERT INTO "projects"("id", "name", "description", "usage_limit", "partner_id",
 INSERT INTO "users"("id", "full_name", "short_name", "email", "password_hash", "status", "partner_id", "created_at") VALUES (E'\\363\\311\\033w\\222\\303Ci\\265\\343U\\303\\312\\204",'::bytea, 'Noahson', 'William', '1email1@mail.test', E'some_readable_hash'::bytea, 1, NULL, '2019-02-14 08:28:24.614594+00');
 INSERT INTO "projects"("id", "name", "description", "usage_limit", "partner_id", "created_at") VALUES (E'\\363\\342\\363\\371>+F\\256\\263\\300\\273|\\342N\\347\\014'::bytea, 'projName1', 'Test project 1', 0, NULL, '2019-02-14 08:28:24.636949+00');
 INSERT INTO "project_members"("member_id", "project_id", "created_at") VALUES (E'\\363\\311\\033w\\222\\303Ci\\265\\343U\\303\\312\\204",'::bytea, E'\\363\\342\\363\\371>+F\\256\\263\\300\\273|\\342N\\347\\014'::bytea, '2019-02-14 08:28:24.677953+00');
+INSERT INTO "project_members"("member_id", "project_id", "created_at") VALUES (E'\\363\\311\\033w\\222\\303Ci\\265\\343U\\303\\312\\204",'::bytea, E'\\022\\217/\\014\\376!K\\023\\276\\031\\311}m\\236\\205\\300'::bytea, '2019-02-13 08:28:24.677953+00');
```
Comment (egonelbre) [inline satellite/satellitedb/testdata/postgres.v50.sql]: Wouldn't having the new data for only before the last migration been sufficient?

Comment (crawter) [reply satellite/satellitedb/testdata/postgres.v50.sql]: let me check it

File: satellite/satellitedb/migrate.go (hunk 1 above)
Comment (egonelbre) [reply satellite/satellitedb/migrate.go]: PS: same comment about the other statements as well.

**Pull request merged**